tests/bootstrap.php

```diff
@@ -14,3 +14,12 @@
 EOT
     );
 }
+
+\bitExpert\Slf4PsrLog\LoggerFactory::registerFactoryCallback(function ($channel) {
+    if (!\Monolog\Registry::hasLogger($channel)) {
```

Comment:
I'm not used to seeing loggers used in the test suite. What's the objective?

Reply:
Good question

Reply:
I guess it's just been that way in most of the projects that I've worked on, which is not a good reason :)

It has occasionally helped me debug works-on-my-machine-but-not-on-CI issues.

Verbose logs from testing on CI can be forensically analyzed to study when regressions in performance or correctness were introduced. I have only rarely done this.

What are your thinking? Should we leave it at-is? Should we switch the test bootstrap to use a PSR NullLogger and remove the build targets? Something else?

Reply:
I'm fine with trying it, we just want to make sure the logs don't get committed to the repo.

Reply:
I'll just merge this and make the appropriate tweaks.